spec/data-model/README.md

```diff
@@ -110,18 +110,16 @@ interface CatchallKey {
 ## Patterns
 
 Each `Pattern` represents a linear sequence, without selectors.
-Each element of the `body` array MUST either be a non-empty string or an `Expression` object.
+Each element of the `Pattern` MUST either be a non-empty string or an `Expression` object.
```

Comment:
I think this is a good change, especially since `body` rhymes with `complex-body`, which means something entirely else.